packages/ng/src/app/select/select.component.scss

```diff
@@ -0,0 +1,80 @@
+@import "node_modules/@lucca-front/scss/src/theming.overridable";
+@import "node_modules/@lucca-front/icons/src/mixins";
+
+:host {
+	cursor: pointer;
+	padding: 0;
+}
+
+.select-value {
+	color: _color("text.placeholder");
+	line-height: 1;
+	height: 2rem;
+	overflow: hidden;
+	padding: _component("textfield.input.padding-vertical") 1.5rem _component("textfield.input.padding-vertical") _component("textfield.input.padding-horizontal");
+  text-overflow: ellipsis;
+	transition: all _theme("commons.animations.durations.standard") ease;
+	white-space: nowrap;
+
+	&::after {
+		bottom: .55rem;
+		color: _color("text.light");
+		@include makeIcon("chevron_south");
+		font-size: .75rem;
+		pointer-events: none;
+		position: absolute;
+		right: _component("textfield.input.padding-horizontal");
+	}
+}
+
+
+// STATES
+// ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
+
+// Value
+:host-context(.textfield-input.is-filled) {
+	.select-value {
+		color: _color("text.default");
+	}
+}
+
+// Material
+:host-context(.textfield.mod-material) {
+	.select-value {
+		opacity: 0;
```

Comment:
This line hides the carret when the input is empty as the carret is handle by the its `::after`.

Reply:
normally fixed by @jeremie-lucca last commit